lib/queuePopulator/KafkaLogConsumer/LogConsumer.js

```diff
@@ -42,13 +44,17 @@ class LogConsumer {
     setup(done) {
         // partition offsets will be managed by kafka
         const consumerParams = {
-            'enable.auto.offset.store': true,
+            // Manually manage storing offsets to ensure they are only stored
+            // after the batch processing is fully completed.
```

Comment:
bonus issue: this will raise the issues we had (and fixed) in BackbeatConsumer around handling of rebalance, i.e. the need to delay the rebalance until the batch has been completed and offsets commited accordingly, and the need to detect/handle "slow tasks" which could block all further processing... It may be done in another PR (or maybe we could reuse BackbeatConsumer for KafkaLogConsumer?), but we probably can't go to prod without that kind of things...